instruction.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" type="text/css" href="instruction.css" crossorigin="anonymous">
+    <title>Instruction</title>
+</head>
+<center>
+<body bgcolor="lightgreen">
+    <h1>
+        <strong>Instruction</strong>
+    </h1>
+
+    <a href="index.html"><img src="images/back.png" class="leftarrow" style="width:50px;height:50px;"></a>
+
+    <div id="div1">
+    <table id="table1">
+            <tr>
+                <th>Gesture</th>
+                <th>Action</th>
+            </tr>
+    </table>
+    </div>
+            
+    <div id="div2">
+        <table id="table2">
+            <tr>
+                <td>Pinch<img src="images/pinch.png"  style="width:30px;height:30px;"></td>

```

Comment:
Since these images all have the same style, create a class called "gesture_image" and add this class to all these images, then give that class this styling by putting this in the CSS:

.gesture_image {
    width: 30px;
    height: 30px;
}